adaptive-media-web/src/main/resources/META-INF/resources/adaptive_media/view.jsp

```diff
@@ -258,6 +256,57 @@ PortletURL portletURL = renderResponse.createRenderURL();
 	}
 </aui:script>
 
+<aui:script require="metal-dom/src/dom">
+	var dom = metalDomSrcDom.default;
+
+	Liferay.on('start_optimizing', (event) => {
+		let uuid = event.uuid;
+
+		let optimizeIcon = dom.toElement('#<portlet:namespace />icon-optimize-' + uuid);
+		let disableIcon = dom.toElement('#<portlet:namespace />icon-disable-' + uuid);
+
+		<portlet:namespace />disableIcon(optimizeIcon);
+		<portlet:namespace />disableIcon(disableIcon);
+
+	});
+
+	Liferay.on('finish_optimizing', (event) => {
```

Comment:
Same as with the other `Liferay.on`... sounds like this might better fit inside the component